src/package.json

```diff
@@ -85,16 +85,28 @@
       }
     ]
   },
-  "repository": {
-    "type": "git",
-    "url": "git+https://github.com/q-masters/vsqlik.git"
-  },
-  "author": "phl-coding",
-  "license": "MIT",
-  "bugs": {
-    "url": "https://github.com/q-masters/vsqlik/issues"
-  },
-  "homepage": "https://github.com/q-masters/vsqlik",
+  "scripts": {
+    "copyhtml": "copyfiles ./projects/extension/**/*.html ./out",
+    "copyreadme": "copyfiles ../README.md .",
+    "clear": "rimraf ./out",
+    "compile": "npm run build -- --skipLibCheck",
+    "watch": "npm run build -- -watch -p ./",
+    "prebuild": "npm run clear && npm run copyhtml",
```

Comment:
if we remove copy html script we could remove this here too, npm run clear is still good

Reply:
@r-hannuschka please can you remove this and make a commit to the branch and test if this still work, that would cleanup the big scripts section :-)